CHANGELOG.md

```diff
@@ -140,6 +140,48 @@ The library has been expanded to include the v2 service.
 consider this a breaking change, as Bazel 5.x has been in maintenance mode for
 more than 6 months.
 
+**CMake Proto Libraries**: We no longer compile certain protos, unless the
```

Comment:
Maybe "we only compile the service-specific protos if the corresponding client library is enabled via `-DGOOGLE_CLOUD_CPP_ENABLE" ?

Reply:
Done.